packages/common/client-utils/package.json

```diff
@@ -41,7 +41,10 @@
 		"build:compile": "fluid-build . --task compile",
 		"build:docs": "fluid-build . --task api",
 		"build:esnext": "tsc-multi --config ../../../common/build/build-common/tsc-multi.esm.json",
-		"build:test": "tsc-multi --config ./tsc-multi.test.json",
+		"build:test": "concurrently npm:build:test:mocha npm:build:test:jest npm:build:test:types",
+		"build:test:jest": "tsc --project ./src/test/jest/tsconfig.json",
+		"build:test:mocha": "tsc --project ./src/test/mocha/tsconfig.json",
+		"build:test:types": "tsc --project ./src/test/types/tsconfig.json",
 		"check:are-the-types-wrong": "attw --pack",
```

Comment:
Also missing from before or had been picked up by something else?

Reply:
When using tsc-multi, I used the tsc-multi config file to point to all the test tsconfigs (see here: https://github.com/microsoft/FluidFramework/pull/18822/files#diff-16280cc42719ada6f4afe2a14dc7d76e4c84772553cb787770b1fba47566f6c7). This reverts it to what it was before I used tsc-multi.

Reply:
I don't see the test/types config in the before list.?

Reply:
it's definitely there